pyrightconfig.testcases.json

```diff
@@ -6,12 +6,21 @@
         "stubs/**/@tests/test_cases"
     ],
     "typeCheckingMode": "strict",
+    // Extra strict settings
+    "reportShadowedImports": "error", // Don't accidentally name a file something that shadows stdlib
+    "reportCallInDefaultInitializer": "error",
+    "reportImplicitStringConcatenation": "error",
+    "reportPropertyTypeMismatch": "error",
```

Comment:
Same comment here — the (anti-)pattern this lint guards against is often a bad idea, but we wouldn't use it in a test case unless we had a specific reason to do so, in which case this lint would just be annoying

Reply:
Updated and added a comment with similar wording as the one for the other two configs.